tests/test_plotting.py

```diff
@@ -51,3 +53,13 @@ def test_plot_NPclassifier_result(spectra_match_data_path_web_api_error):
     plot = MoleculePlotter.plot_NPclassifier_result(spectra_match_data_path)
 
     assert isinstance(plot, matplotlib.container.BarContainer)
+
+
+def test_mirror_plot(small_mgf_path):
```

Comment:
I think you need to merge target branch into yours. This fixture name is not the same anymore

Reply:
ok